src/test/components/ButtonWithPanel.test.js

```diff
@@ -52,6 +52,7 @@ describe('shared/ButtonWithPanel', () => {
     expect(container.firstChild).toMatchSnapshot();
   });
 
+  /* These tests will be rewritten when ConfirmDialog is introduced.
   describe('various panel contents', () => {
```

Comment:
I think it would be better to skip them rather than comment them out. It would need an eslint-disable comment to do so, but we get a notification for skipped tests. If flow complains, I would comment out the internals.

Reply:
actually I think we also now have an eslint rule for commented tests :-) (that gives warnings only) => https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/no-commented-out-tests.md
Initially I was re-enabling in a later commit in the same commit, that I later removed. I delete them in one of my next PR so I can as well do it here too.